csi.proto

```diff
@@ -1213,6 +1213,13 @@ message NodeGetVolumeStatsRequest {
   // the process serving this request.
   // This is a REQUIRED field.
   string volume_path = 2;
+
+  // The path where the volume is staged, if the plugin has the
+  // STAGE_UNSTAGE_VOLUME capability, otherwise empty.
+  // If not empty, it MUST be an absolute path in the root
+  // filesystem of the process serving this request.
+  // This field is OPTIONAL.
+  string staging_path = 3;
```

Comment:
Thank you very much for this change!
If we want to remain consistent with other places this is used, its actually called: `staging_target_path`
eg:
https://github.com/container-storage-interface/spec/blob/9e773d2dd46a9b113d94fb3c245231d231bcfeda/csi.proto#L1152

https://github.com/container-storage-interface/spec/blob/9e773d2dd46a9b113d94fb3c245231d231bcfeda/csi.proto#L1129

Reply:
I like your suggestion, I have implemented it.

Reply:
I thought this was going to be renamed from staging_path to staging_target_path for consistency .. what happened?